### DISCUSSION TITLE
Akka/Pekko Actors

### DISCUSSION THREAD

**Discussion by andreilgeorgescu** [riverwalk-software/astro-test#80] [Q&A] [answered]
@daniel-ciocirlan I see it is called both `Akka/Pekko Actors` and `Akka/Pekko Typed Essentials` in different parts of the website.

Is it an "Essentials" course similar to the scala, kotlin, and spark essentials courses?

**Comment by daniel-ciocirlan** [comment, marked as answer]
Legit question. Let's call it "Akka/Pekko Essentials".
